src/lib/modules/module/module_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 	"time"
 
-	config "github.com/bench-routes/bench-routes/src/lib/config_v2"
+	config "github.com/bench-routes/bench-routes/src/lib/config"
 	"github.com/stretchr/testify/require"
 )
 
```

Comment:
We need to update the New() function and also the respective test config file with the recent changes

Reply:
Please cover this in your next commit.